build_tools/convert-python-script-to-notebook.py

```diff
@@ -0,0 +1,138 @@
+import sys
```

Comment:
Maybe a small docstring to explain what is going on in this file / what is its purpose

Reply:
Yeah that's a good point, I added small docstrings (be it only to remember it myself) to the function but not to the module.

Reply:
Done in https://github.com/INRIA/scikit-learn-mooc/commit/d61c5c1cb2f4bcd2ee443005b69d42831eabdaa8